.github/workflows/workflow.yml

```diff
@@ -17,5 +17,8 @@ jobs:
     - name: Build
       run: make all
 
+    - name: Unit testing
+      run: ./build/test/unit/test -c
```

Comment:
Why isn't this run via make check? If we want to have separate unit and integration tests, they can have separate targets (make unittest; make integrationtest), which are combined via make check.

Reply:
ack